src/librustc_metadata/locator.rs

```diff
@@ -845,6 +860,15 @@ fn get_metadata_section_imp(target: &Target,
                 Ok(blob)
             }
         };
+    } else if flavor == CrateFlavor::Rmeta {
+        let mut file = File::open(filename).map_err(|_|
+            format!("could not open file: '{}'", filename.display()))?;
```

Comment:
I was basically following the errors from the rlib branch directly above here. Those follow the same pattern (ignoring the actual error), although in that case because the error gets converted into an `Option` in the meantime. So I could (c.f., rlibs) add the extra info here, but it would be inconsistent with the rlib version.